autogpt_platform/frontend/src/components/CustomNode.tsx

```diff
@@ -31,7 +31,7 @@ import {
   parseKeys,
   setNestedProperty,
 } from "@/lib/utils";
-import { Button } from "@/components/ui/button";
+import { Button } from "@/components/atoms/Button/Button";
```

Comment:
does this fit in with the rest of the UI?

Reply:
yes it does, there is another comment  on this page where @0ubbe tells me specifically to use this

Reply:
We want to avoid using the old components on the new UI as much as possible, otherwise we would never migrate away from the 2 old component libraries and the UX will stay fragmented 😮‍💨